lib/openstudio/extension/runner.rb

```diff
@@ -41,13 +41,14 @@ def initialize(dirname = Dir.pwd, bundle_without = [], options = {})
 
         # if the dirname contains a runner.conf file, then use the config file to specify the parameters
 
-        @options = OpenStudio::Extension::RunnerConfig.default_config
+        @options = OpenStudio::Extension::RunnerConfig.default_config(dirname)
         # ORDER of PRECEDENCE: default config < runner.conf file < options passed in directly
         if File.exist?(File.join(dirname, OpenStudio::Extension::RunnerConfig::FILENAME))
           puts 'Using runner options from runner.conf file'
           runner_config = OpenStudio::Extension::RunnerConfig.new(dirname)
-          # use the default values overriden with runner.conf values
-          @options = @options.merge(runner_config.options)
+          # use the default values overriden with runner.conf values where not
+          # nil nor empty strings
+          @options = @options.merge(runner_config.options.reject{|k, v| v.nil? || (v.kind_of?(String) && v === '')})
```

Comment:
Reject empty (nil or empty strings) from runner.conf when merging.

Reply:
Note that line 65 is technically not needed anymore now... but I was wary of removing it